app/controllers/api/visioplainte/creneaux_controller.rb

```diff
@@ -4,6 +4,8 @@ class Api::Visioplainte::CreneauxController < Api::Visioplainte::BaseController
   before_action :validate_date_debut
   before_action :validate_date_range, only: [:index]
 
+  prepend_before_action :allow_authentication_with_read_only_api_key
```

Comment:
```suggestion
  prepend_before_action :allow_authentication_with_read_only_api_key, only: [:index, :prochain]
```

éventuellement pour se prémunir d’un éventuel oubli futur lors d’un ajout d’endpoint de modification (même si peu probable sur ce contrôleur)

Reply:
bien vu !